source/development/coredump.rst

```diff
@@ -0,0 +1,195 @@
+        :class: output                            
+
+        │|/lib/systemd/systemd-coredump %P %u %g %s %t                                                                                                                                                                                                                            
+
+#. **Enable Core Dump Collection**:
+    Enable core dump collection by setting the ``Storage=`` option in the systemd ``coredump.conf`` file. You can set it to ``external`` to store core dumps externally, or ``none`` to disable core dump collection altogether.
+
+    .. code-block:: console
+
+        systemctl edit systemd-coredump
+
+    Add the following lines to the editor that opens:
+
+    .. code-block:: console
+
+        [Coredump]
+        Storage=external
+
+#. **Recommended** – Set a size limit for core dump files. For example, 2 GB.
+
+    .. code-block:: console
+
+        ProcessSizeMax=2G
+
+#. **Restart systemd-coredump Service**:
+    After making changes to the configuration, restart the systemd-coredump service for the changes to take effect:
+
+    .. code-block:: console
+
+        systemctl restart systemd-coredump
+
+By following these steps, you can set up core dump generation using systemd. This allows for centralized management and configuration of core dumps across your system.
+
+To disable core dump generation, you can modify the configuration of the systemd-coredump service. Here's how you can do it:
+
+#. **Edit systemd-coredump Configuration**:
+    Open the systemd-coredump configuration file for editing:
+
+    .. code-block:: console
+
+        systemctl edit systemd-coredump
+
+#. **Add Configuration to Disable Core Dumps**:
+    Add the following lines to the editor that opens:
+
+    .. code-block:: console
+
+        [Coredump]
+        Storage=none
+
+    This configuration sets the ``Storage`` option to ``none``, effectively disabling core dump storage.
+
+#. **Restart systemd-coredump Service**:
+    After making changes to the configuration, restart the systemd-coredump service for the changes to take effect:
+
+    .. code-block:: console
+
+        systemctl restart systemd-coredump
+
+#. **Verify Configuration**:
+    Check the status of the systemd-coredump service to ensure it is running without errors:
+
+    .. code-block:: console
+
+        systemctl status systemd-coredump
+
+Once you've completed these steps, core dump generation will be disabled on your system. Any attempts to generate core dumps will not result in files being stored.
+
+Manual configuration
+^^^^^^^^^^^^^^^^^^^^
+
+Setting up core dump generation without using systemd involves configuring the operating system's core dump settings manually. Here's how you can do it on Linux systems:
+
+#. **Identify the Target Directory for Core Dumps**:
```

Comment:
Add an example. Do we mean `/var/core/` as an example? We could move this step down before enabling core dump generation since it seems to be related.

Reply:
This comment is redundant, the configuration is explained below.
the point will be removed